python/ray/tests/test_signal.py

```diff
@@ -30,6 +35,22 @@ def receive_all_signals(sources, timeout):
         else:
             results.extend(r)
 
+@pytest.fixture()
+def two_node_cluster():
+    internal_config = json.dumps({
+        "initial_reconstruction_timeout_milliseconds": 200,
+        "num_heartbeats_timeout": 10,
+    })
+    cluster = ray.tests.cluster_utils.Cluster()
```

Comment:
It might not cause any issue, but better pass `internal_config`  to the head node as well to make sure all nodes in the cluster have the same config.